amundsen_application/api/mail/v0.py

```diff
@@ -7,24 +7,35 @@
 from flask.blueprints import Blueprint
 
 from amundsen_application.api.utils.notification_utils import get_notification_content
+from amundsen_application.api.utils.request_utils import get_query_param
 from amundsen_application.log.action_log import action_logging
 
 LOGGER = logging.getLogger(__name__)
 
 mail_blueprint = Blueprint('mail', __name__, url_prefix='/api/mail/v0')
 
 
-@mail_blueprint.route('/feedback', methods=['POST'])
-def feedback() -> Response:
-    """ An instance of BaseMailClient client must be configured on MAIL_CLIENT """
+class MailClientNotImplemented(Exception):
```

Comment:
We have logic for raising a "not implemented" error in other places. How about we put this in a new `amundsen_application/api/exceptions.py` can call it `NotImplementedException`. After it's merged down to master we can then use it other places as well.

Reply:
:+1: